presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -867,6 +868,17 @@ protected Void visitRenameColumn(RenameColumn node, Integer context)
         }
 
         @Override
+        protected Void visitDropColumn(DropColumn node, Integer context)
+        {
+            builder.append("ALTER TABLE ")
+                    .append(node.getTable())
```

Comment:
Use `formatName` so that we properly quote names with special characters.

(Much of `SqlFormatter` is broken in this regard and needs to get cleaned up with appropriate tests. But we should make sure to do it right when adding new stuff.)